specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
+
+The user requesting the split trigger an automatic vote to approve the split. If the request is approved then the original requestor will also be the first leader of the new community by default. 
+ 
+After splitting the community, then each member should be given a choice as to which of the resulting communities they wish to belong to, or if they wish to be a member of both.
 
-List all users and their types
+### Merge two Communities
+We don't anticipate this to happen often, but it could be the case that two communities have shrinking membership and would like to merge to increase the total membership in one Community.
 
-### Actions
+A merge proposal should be approved by majority vote in both communities before it is allowed to happen.
 
-List the actions each collective will take individually.
+It's not easy to design rules for how the merge should be executed, based on the complexity of possible permutations of the pre-merge Communities. It seems plausible that merge power should not be given to regular users but could be given as an admin power to a dev / superuser. The alternative would be that merging not be supported, in which case the community will manually have to move from and abandon one community in favor of the other.
 
-Describe which action is done by which actors. Feel free to use a table format or provide your own graphics or bundle the description in the Actions section. A "swimlane process chart" often works well here.
+### Interact with my Community
+Members should at minimum be able to post messages within the community which become publicly readable to anyone regardless of membership status, and where other members are allowed to comment on the message.
 
-|         | Action 1 | Action 2 | Action 3 | action 4 |
+Messages posted on boards that I follow should show up in my personal feed in chronological order. But they don't show up in the feed of users who are not members and don't follow the community.
+
+Members or admins (depending on Community settings) should be able to post polls for the community, which only community members are allowed to vote in.
+
+As a member I should be able to see all other members listed in a sidebar, along with their membership / admin status.
+
+### Ban a user from a Community
+Banning is a strong power and should be used very restrictively. We suggest milder and more human forms of restricting users that are not following the code of conduct for a community. 
+- Issue a warning for breach of code of conduct
+- Temporarily silence the user for a day
+- Temporarily silence the user for a week
+- Silence the user until un-silenced (perhaps after taking some corrective action)
+- Banning the user with chance of reapplying
+- Banning the user with chance of reapplying (permanently)
```

Comment:
what's the difference between the 2 cases above?

Reply:
Missing a word "no" in the last one.